File: VB/Person.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic

Public Class Person
	Public Shared Function GetDemoData() As List(Of Person)
		Dim list As New List(Of Person)()
		list.Add(New Person() With {.ID = 1, .FirstName = "John", .LastName = "Doe"})
		list.Add(New Person() With {.ID = 2, .FirstName = "Jane", .LastName = "Doe"})
		Return list
	End Function

	Public Sub New()
	End Sub

	Private _ID As Integer
	Public Property ID() As Integer
		Get
			Return _ID
		End Get
		Set(ByVal value As Integer)
			_ID = value
		End Set
	End Property


	Private _FirstName As String
	Public Property FirstName() As String
		Get
			Return _FirstName
		End Get
		Set(ByVal value As String)
			_FirstName = value
		End Set
	End Property

	Private _LastName As String
	Public Property LastName() As String
		Get
			Return _LastName
		End Get
		Set(ByVal value As String)
			_LastName = value
		End Set
	End Property
End Class

File: CS/Person.cs
using System;
using System.Collections.Generic;

public class Person {
    public static List<Person> GetDemoData() {
        List<Person> list = new List<Person>();
        list.Add(new Person() { ID = 1, FirstName = "John", LastName = "Doe" });
        list.Add(new Person() { ID = 2, FirstName = "Jane", LastName = "Doe" });
        return list;
    }

    public Person() {
    }

    private int _ID;
    public int ID {
        get { return _ID; }
        set {
            _ID = value;
        }
    }
    

    private string _FirstName;
    public string FirstName {
        get { return _FirstName; }
        set {
            _FirstName = value;
        }
    }

    private string _LastName;
    public string LastName {
        get { return _LastName; }
        set {
            _LastName = value;
        }
    }
}